manifests/a/A-SIT/PDF-Over/4.4.5/A-SIT.PDF-Over.installer.yaml

```
@@ -15,6 +15,6 @@ AppsAndFeaturesEntries:
 Installers:
 - Architecture: x64
   InstallerUrl: https://technology.a-sit.at/download/pdf-over-win/?version=4.4.5
-  InstallerSha256: B99D4DAB49F08E7FCC19A329CE6D564BC963C0EC40534CA92ABCBA98D6227021
+  InstallerSha256: 6A5CC6EB07AB0E6AAE27832B21CA7829983F456A6EF1730527CDD03757631685

```

1/3/2025 6:39:44 PM
Installer Verification Analysis Context Information: 
Specified hash doesn't match.
Actual hash:   B99D4DAB49F08E7FCC19A329CE6D564BC963C0EC40534CA92ABCBA98D6227021
Expected hash: 6a5cc6eb07ab0e6aae27832b21ca7829983f456a6ef1730527cdd03757631685

```suggestion
  InstallerSha256: B99D4DAB49F08E7FCC19A329CE6D564BC963C0EC40534CA92ABCBA98D6227021
```